composeApp/src/commonMain/kotlin/band/effective/office/elevator/ui/booking/components/OptionMenu.kt

```diff
@@ -0,0 +1,148 @@
+package band.effective.office.elevator.ui.booking.components
+
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.selection.selectable
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import band.effective.office.elevator.MainRes
+import band.effective.office.elevator.ui.models.TypesList
+import dev.icerock.moko.resources.compose.painterResource
+import dev.icerock.moko.resources.compose.stringResource
+
+@Composable
+fun OptionMenu(
+    isExpandedCard: Boolean,
+    isExpandedOptions: Boolean,
+    onClickOpenBookPeriod: () -> Unit
+) {
+    Column {
+        AnimatedVisibility(visible = isExpandedCard) {
+            Column(
+                modifier = Modifier.fillMaxWidth().padding(top = 16.dp)
+                    .background(MaterialTheme.colors.onBackground)
+            ) {
+                Row(
+                    horizontalArrangement = Arrangement.Center,
+                    modifier = Modifier.fillMaxWidth()
+                ) {
+                    Image(
+                        modifier = Modifier.padding(vertical = 20.dp),
+                        painter = painterResource(MainRes.images.icon_map), //TODO(Заменить карту!")
```

Comment:
Who write TODO? We write author in todo and use English
Example: //TODO(Maksim Mishenko): not implement